### ISSUE TITLE
Performance is lower than the results you report.

### ISSUE THREAD

**Issue by zhouyuan888888** [YudeWang/deeplabv3plus-pytorch#48]
By using the default config you provide, I have achieved 79.206 % (your result is 79.916 %) based on deeplabv3+res101 on the pascalvoc validation set. Can you tell me why this happened? By the way:
1) I have used the pretrained res-101.
2) My environment is python3.6, cuda8.0, pytorch0.4.1, 
3) The augment pascal voc dataset is downloaded from DrSleep
4) I re-train the model without any modifications for the code you provide.

How can I do to achieve the performance you report in the README.? Thank you for your help~

**Comment by YudeWang** [comment]
@zhouyuan888888 
Have you use deeplabv3+vocfinetuning in experiment folder to refine your 79.2.6% model? The annotation of VOC train set is better than trainaug set. Experiment of deeplabv3+vocfinetuning will finetune on VOC train set with a small learning rate and further improve mIoU.

**Comment by zhouyuan888888** [comment]
Thank you for your reply : ). I know finetuning the model on the original pascal voc train set can improve accuracy. But in README you pointed out that the performance of 79.155% and 79.916 % is achieved without finetuning. So I have tried many times to reproduce your results but failed. T_T, So, if I want to reproduce your results, what can I do? It is so strange because I have not modified any your codes or configurations.

Thank you so much for your answering! :)

**Comment by YudeWang** [comment]
@zhouyuan888888 
deeplabv3+res101 achieve 79.155% and deeplabv3+xception achieve 79.945%
I find you have said that pretrained res101 model was used. The comparison is correct? your-79.206%>my-79.155%

**Comment by zhouyuan888888** [comment]
Yes, I used the pretrained res101 model. So I compare our results based on the res101 version. 79.206 % and 79.916 % are all evaluated by using multi-scale and flip strategies. :)

**Comment by YudeWang** [comment]
@zhouyuan888888 
Maybe it is caused by random seed? The code I released does not fix it. And what about single scale test?

**Comment by zhouyuan888888** [comment]
I think the gap about 0.7% is a little large. For single-scale test without flipping, I only got about 78.014 % MIoU (yours is about 79.155% ) T_T. By the way, have you ever used any other tracks to further import accuracy?

You are so kind, thank you for your quick reply!!!

**Comment by YudeWang** [comment]
@zhouyuan888888 
There is no other tricks. Please finetuning on train set (the paper includes that), or select better super-parameters by yourself. Dense CRF also can be considered.

**Comment by JingyuLi-code** [comment]
> By using the default config you provide, I have achieved 79.206 % (your result is 79.916 %) based on deeplabv3+res101 on the pascalvoc validation set. Can you tell me why this happened? By the way:
> 
> 1. I have used the pretrained res-101.
> 2. My environment is python3.6, cuda8.0, pytorch0.4.1,
> 3. The augment pascal voc dataset is downloaded from DrSleep
> 4. I re-train the model without any modifications for the code you provide.
> 
> How can I do to achieve the performance you report in the README.? Thank you for your help~

Hello, 我和你上述保持同样的参数，包括pretrained res-101.augment pascal voc dataset is downloaded from DrSleep，以及 多尺度测试和裁剪,在4块RTX2080Ti实验。
在训练46个epoch]后测试，得到以下结果
  backbound: 94.368%      aeroplane: 92.200%
    bicycle: 43.120%           bird: 89.926%
       boat: 71.572%         bottle: 82.099%
        bus: 95.505%            car: 89.099%
        cat: 92.570%          chair: 40.066%
        cow: 89.992%    diningtable: 56.415%
        dog: 87.355%          horse: 86.244%
  motorbike: 87.790%         person: 86.892%
pottedplant: 62.389%          sheep: 88.140%
       sofa: 47.990%          train: 87.240%
  tvmonitor: 78.863%    
======================================================
       mIoU: 78.564%

在46epoch 我没得到满意的结果。所以我尝试增大epoch,保持github其他参数不变，将epoch改为60，训练60个epoch后测试：
  backbound: 94.560%      aeroplane: 93.138%
    bicycle: 42.715%           bird: 89.845%
       boat: 76.035%         bottle: 80.670%
        bus: 95.354%            car: 89.574%
        cat: 94.906%          chair: 41.242%
        cow: 87.774%    diningtable: 59.855%
        dog: 89.180%          horse: 86.608%
  motorbike: 86.478%         person: 87.563%
pottedplant: 61.951%          sheep: 89.305%
       sofa: 47.412%          train: 89.707%
  tvmonitor: 77.503%    
======================================================
       mIoU: 79.113%
在itr35000测试： 
  backbound: 94.661%      aeroplane: 93.178%
    bicycle: 42.561%           bird: 89.823%
       boat: 76.099%         bottle: 80.941%
        bus: 95.403%            car: 89.597%
        cat: 94.806%          chair: 40.680%
        cow: 87.209%    diningtable: 60.366%
        dog: 88.884%          horse: 86.239%
  motorbike: 86.410%         person: 87.537%
pottedplant: 62.798%          sheep: 89.348%
       sofa: 49.260%          train: 90.454%
  tvmonitor: 77.807%    
======================================================
       mIoU: 79.241%
然而，在vocfinetune 训练60epoch结果会很差：
finetune
  backbound: 93.994%      aeroplane: 91.886%
    bicycle: 43.208%           bird: 89.967%
       boat: 71.949%         bottle: 78.653%
        bus: 94.821%            car: 88.435%
        cat: 94.454%          chair: 38.411%
        cow: 88.571%    diningtable: 45.917%
        dog: 88.607%          horse: 86.051%
  motorbike: 85.921%         person: 87.293%
pottedplant: 60.941%          sheep: 87.682%
       sofa: 44.132%          train: 85.685%
  tvmonitor: 76.548%    
======================================================
       mIoU: 77.292%

所以，请问你后续有没有进行超参数的调试以达到工程所达到的结果？thanks!

**Comment by CharlesPikachu** [comment]
maybe you can try this: https://github.com/SegmentationBLWX/sssegmentation
